File: BikeManagerAPI/BikeManagerAPI/Models/Estoque.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;

namespace BikeManagerAPI.Models
{
    public partial class Estoque
    {
        public int CdEstoque { get; set; }
        public int CdProduto { get; set; }
        public int? CdOrdemServico { get; set; }
        public int? CdNotaEntrada { get; set; }
        public decimal QtProduto { get; set; }
        public decimal VlCusto { get; set; }
        public decimal? VlCustoMedio { get; set; }
        public string TpLancamento { get; set; }
        public DateTime DtRegistro { get; set; }
        public DateTime? DtAlteracao { get; set; }
        public NotaEntrada NotaEntrada { get; set; }
        public OrdemServico OrdemServico { get; set; }
    }
}

File: WebApp/src/app/cadastros/models/estoque.ts
import { NotaEntrada } from "./notaEntrada";
import { OrdemServico } from "./ordemServico";

export class Estoque {
    cdEstoque: number;
    cdProduto: number;
    cdOrdemServico: number;
    cdNotaEntrada: number;
    qtProduto: number;
    tpLancamento: string;
    vlCusto: number;
    qtAtual: number;
    vlCustoMedio: number;
    dtRegistro: Date;
    dtAlteracao: Date;
    notaEntrada: NotaEntrada;
    ordemServico: OrdemServico;
}